Assets/VRTK/Scripts/Locomotion/VRTK_GazeDrivenLocomotion.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Gaze Driven Movement|Scripts|0070
+namespace VRTK
+{
+    using UnityEngine;
+    using UnityEngine.UI;
+    /// <summary>
+    /// The Gaze Driven Movement script creations a directional "joystick" underneath a given game object within the seen. Movement is driven by moving the joystick using the HMD to drag across the gamepad.
+    /// </summary>
+    /// <remarks>
+    /// This script is meant to be used for 3rd person VR games.
+    /// Attach this script to the `[CameraRig]` prefab. Attach the VRTK_DirectionPad prefab to get the UI for the directional pad.
+    /// </remarks>
```

Comment:
As there is also an example for this, include the `<example></example>` tag underneath the remarks describing where the example can be found.